src/lib/Libifl/pbsD_connect.c

```diff
@@ -171,6 +171,45 @@ get_hostsockaddr(char *host, struct sockaddr_in *sap)
 	return -1;
 }
 
+/**	
+ * @brief	
+ *	-hostnmcmp - compare two hostnames, allowing a short name to match a longer	
+ *	version of the same	
+ *	
+ * @param[in] s1 - hostname1	
+ * @param[in] s2 - hostname2	
+ *	
+ * @return	int	
+ * @retval	1	success	
+ * @retval	0	failure	
+ *	
+ */	
+static int	
+hostnmcmp(char *s1, char *s2)	
+{	
+	/* Return failure if any/both the names are NULL. */	
+	if (s1 == NULL || s2 == NULL)	
+		return 1;	
+#ifdef WIN32	
```

Comment:
I think Windows code is not needed (here and other places)

Reply:
Why do you think? Windows command would still use pbs_connet() IFL to Linux server.

Reply:
oops, my bad, that's true